### ISSUE TITLE
How to activate collab's gpu to train mask rcnn model

### ISSUE THREAD

**Issue by Yaseen0361** [matterport/Mask_RCNN#2902]
I have been working on this model for couple of weeks but still facing the issues that tensorflow library that works with mrcnn is not supported by google colab due to which I am unable to access the free gpu and it takes a complete day to train 10 epochs of 10 steps. 
how will I activate the gpu runtime with tensorflow 1.13.1 version(supported by mrcnn model)
`use_multiprocessing=True` and multiple workers may duplicate your data. Please consider using the`keras.utils.Sequence class.
  UserWarning('Using a generator with `use_multiprocessing=True`'
Epoch 1/10
 9/10 [==========================>...] - ETA: 89s - loss: 5.1812 - rpn_class_loss: 0.0661 - rpn_bbox_loss: 2.0932 - mrcnn_class_loss: 1.0029 - mrcnn_bbox_loss: 0.9374 - mrcnn_mask_loss: 1.0817 /usr/local/lib/python3.7/dist-packages/keras/engine/training.py:2142: UserWarning: Using a generator with `use_multiprocessing=True` and multiple workers may duplicate your data. Please consider using the`keras.utils.Sequence class.
  UserWarning('Using a generator with `use_multiprocessing=True`'
10/10 [==============================] - 2200s - loss: 4.9398 - rpn_class_loss: 0.0856 - rpn_bbox_loss: 2.1344 - mrcnn_class_loss: 0.9026 - mrcnn_bbox_loss: 0.8437 - mrcnn_mask_loss: 0.9735 - val_loss: 2.9885 - val_rpn_class_loss: 0.0627 - val_rpn_bbox_loss: 1.0605 - val_mrcnn_class_loss: 0.1267 - val_mrcnn_bbox_loss: 0.8837 - val_mrcnn_mask_loss: 0.8549
Epoch 2/10
10/10 [==============================] - 2080s - loss: 2.8650 - rpn_class_loss: 0.0653 - rpn_bbox_loss: 0.8709 - mrcnn_class_loss: 0.2062 - mrcnn_bbox_loss: 0.9295 - mrcnn_mask_loss: 0.7931 - val_loss: 2.5669 - val_rpn_class_loss: 0.0251 - val_rpn_bbox_loss: 0.7778 - val_mrcnn_class_loss: 0.1389 - val_mrcnn_bbox_loss: 0.8665 - val_mrcnn_mask_loss: 0.7586
Epoch 3/10
10/10 [==============================] - 2071s - loss: 2.5079 - rpn_class_loss: 0.0178 - rpn_bbox_loss: 0.6116 - mrcnn_class_loss: 0.1541 - mrcnn_bbox_loss: 1.0084 - mrcnn_mask_loss: 0.7159 - val_loss: 2.3405 - val_rpn_class_loss: 0.0179 - val_rpn_bbox_loss: 0.7233 - val_mrcnn_class_loss: 0.1208 - val_mrcnn_bbox_loss: 0.7990 - val_mrcnn_mask_loss: 0.6794
Epoch 4/10
10/10 [==============================] - 2076s - loss: 3.5875 - rpn_class_loss: 0.0330 - rpn_bbox_loss: 2.2295 - mrcnn_class_loss: 0.0901 - mrcnn_bbox_loss: 0.6266 - mrcnn_mask_loss: 0.6082 - val_loss: 2.1113 - val_rpn_class_loss: 0.0173 - val_rpn_bbox_loss: 0.6490 - val_mrcnn_class_loss: 0.1017 - val_mrcnn_bbox_loss: 0.7573 - val_mrcnn_mask_loss: 0.5859
Epoch 5/10
10/10 [==============================] - 2081s - loss: 2.0695 - rpn_class_loss: 0.0174 - rpn_bbox_loss: 0.6024 - mrcnn_class_loss: 0.1108 - mrcnn_bbox_loss: 0.7344 - mrcnn_mask_loss: 0.6044 - val_loss: 2.2599 - val_rpn_class_loss: 0.0158 - val_rpn_bbox_loss: 0.6386 - val_mrcnn_class_loss: 0.1036 - val_mrcnn_bbox_loss: 0.8452 - val_mrcnn_mask_loss: 0.6566
Epoch 6/10
10/10 [==============================] - 2073s - loss: 2.1327 - rpn_class_loss: 0.0219 - rpn_bbox_loss: 0.5814 - mrcnn_class_loss: 0.1610 - mrcnn_bbox_loss: 0.6752 - mrcnn_mask_loss: 0.6932 - val_loss: 2.2272 - val_rpn_class_loss: 0.0170 - val_rpn_bbox_loss: 0.6450 - val_mrcnn_class_loss: 0.1082 - val_mrcnn_bbox_loss: 0.8118 - val_mrcnn_mask_loss: 0.6452
Epoch 7/10
10/10 [==============================] - 2075s - loss: 3.2746 - rpn_class_loss: 0.0522 - rpn_bbox_loss: 1.5602 - mrcnn_class_loss: 0.1547 - mrcnn_bbox_loss: 0.8337 - mrcnn_mask_loss: 0.6737 - val_loss: 2.0564 - val_rpn_class_loss: 0.0172 - val_rpn_bbox_loss: 0.6224 - val_mrcnn_class_loss: 0.1280 - val_mrcnn_bbox_loss: 0.6824 - val_mrcnn_mask_loss: 0.6063
Epoch 8/10

**Comment by Densus100** [comment]
Nope, you can't use google colab anymore to train this matterport mask r-cnn. This is due to Google Colab removed support for TensorFlow 1 in their latest release of 2022/8/11. You could see more details in https://colab.research.google.com/notebooks/relnotes.ipynb
